setup.py

```diff
@@ -13,7 +13,7 @@
     url='https://github.com/onaio/superset-patchup',
     packages=find_packages(exclude=['docs', 'tests']),
     install_requires=[
-        'apache-superset',
+        'apache-superset==1.4.1',
```

Comment:
Do we want to pin the patch version too? I would have maybe gone with `~=1.4.1`

Reply:
I thought to have this lib to work with the same superset version as its there via ansible, to ensure there are no compatibility issues. One thing I have observed in superset is that do frequent refactoring, leading to changes in method calls.